mysql-client/mysql-liveness-check.sh

```diff
@@ -61,28 +91,13 @@ liveness_check()
 #  MAIN   #
 ###########
 
-# Usage
-if [ "$#" -ne 1 ]; then
-    echo 
-    echo "Usage:"
-    echo
-    echo "$0 <db-credentials.conf>"
-    echo
-    echo "<db-credentials.conf> JSON file with db server,user,password"
-    exit 1 
-fi  
-
-# Parse the database credentials JSON file 
-db=($(jq -r .[] $1))
-
-# DB Server Cred Lookup Table
-
-# db_server_ip : ${db[0]} 
-# db_user      : ${db[1]} 
-# db_password  : ${db[2]}
+# Verify availability of DB credentials
+if [[ -z ${DB_SVC} || -z ${DB_USER} || -z ${DB_PASSWORD} ]]; then
+ usage;
```

Comment:
should we pass an argument for this?

Reply:
It should be set in the ENV (In our context, it is set as pod/container ENV). And since it is required to connect to the service, w/o which liveness wont work, they have been used as mandatory ENV (i.e., no defaults specified unlike other timeout/retry params)

Reply:
I meant argument for usage to print the help, like, uasge --help

Reply:
or we can remove 'if' condition in usage

Reply:
Got it. Will update